styles/globals.scss

```diff
@@ -141,82 +147,18 @@ body {
         max-width: 650px;
       }
     }
-  }
-
-  .shimmer {
```

Comment:
Just making sure: Did you mean to remove all these shimmer styles?

Reply:
yup, at some point (bad merge? who knows) they got fully duplicated - see (old) line 221. Forgot to comment that - sorry!